flink-core/src/main/java/org/apache/flink/configuration/HighAvailabilityOptions.java

```diff
@@ -220,6 +225,50 @@ public class HighAvailabilityOptions {
                                                     + "with unresolvable hostnames.")
                                     .build());
 
+    public static final ConfigOption<Map<String, String>> ZOOKEEPER_CLIENT_AUTHORIZATION =
+            key("high-availability.zookeeper.client.authorization")
+                    .mapType()
+                    .noDefaultValue()
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "Add connection authorization Subsequent calls to this method overwrite the prior calls. "
+                                                    + "In certain cases ZooKeeper requires additional Authorization information. "
+                                                    + "For example list of valid names for ensemble in order to prevent accidentally connecting to a wrong ensemble."
+                                                    + "Each entry of type Map.Entry<String, String> will be transformed "
+                                                    + "into an AuthInfo object with the constructor AuthInfo(String, byte[]). "
+                                                    + "The field entry.key() will serve as the String scheme value, while the field entry.getValue() "
+                                                    + "will be initially converted to a byte[] using the String#getBytes() method with %s encoding"
+                                                    + "If not set the default configuration for a Curator would be applied.",
+                                            text(ConfigConstants.DEFAULT_CHARSET.displayName()))
+                                    .build());
+
+    public static final ConfigOption<Duration> ZOOKEEPER_MAX_CLOSE_WAIT_MS =
+            key("high-availability.zookeeper.client.max-close-wait-ms")
```

Comment:
```suggestion
    public static final ConfigOption<Duration> ZOOKEEPER_MAX_CLOSE_WAIT =
            key("high-availability.zookeeper.client.max-close-wait")
```
The `ms" suffix is not needed anymore because we're relying on `Duration`.